### PR TITLE
Feature create user account

### PR DESCRIPTION
**Pull request by roxuy**
card: 
https://trello.com/c/ikuwACdO/2-5-as-a-user-i-should-be-able-to-sign-up-so-my-account-is-created-in-target-mvd

### CODE REVIEW

File: app/views/devise/passwords/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<h2>Change your password</h2>
+
+<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f| %>
```
Comment (santir489) [inline app/views/devise/passwords/edit.html.erb:3]: We can delete all this html.erb files right? We won't be using them as it is only API

File: db/migrate/20210729044000_alter_gender_to_user.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class AlterGenderToUser < ActiveRecord::Migration[6.1]
```
Comment (enzofab91) [inline db/migrate/20210729044000_alter_gender_to_user.rb:1]: This can be deleted since is empty.

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+DSrequire 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (enzofab91) [inline spec/models/user_spec.rb:4]: Maybe would be nice to add some validation in the model and then a test here for that.

Mail uniqueness or presence. See https://guides.rubyonrails.org/active_record_validations.html

**Review by enzofab91 — COMMENTED** [review]
Good work!

Left some comments.

File: spec/models/user_spec.rb (hunk 3 above)
Comment (roxuy) [reply spec/models/user_spec.rb]: I added some validations and tests. I still have to test the email uniqueness

**Review by santir489 — APPROVED** [review]

**Review by enzofab91 — APPROVED** [review]
Good work 👏

**Pull request merged**